ci/schema/duckdb.sql

```diff
@@ -45,11 +45,14 @@ CREATE OR REPLACE TABLE awards_players (
     notes TEXT
 );
 
+CREATE TYPE enum_t AS ENUM ('alice', 'bob', 'carol');
+
 CREATE OR REPLACE TABLE functional_alltypes (
     "index" BIGINT,
     "Unnamed: 0" BIGINT,
     id INTEGER,
     bool_col BOOLEAN,
+    enum_col enum_t,
```

Comment:
You can't add a column to this table without modifying the data.

I would suggest something like changing the `string_col` column to be an enum of `0 to 9 inclusive, which reflects the data in `ci/ibis-testing-data/functional_alltypes.csv`.